src/hotspot/share/prims/jvmtiEnvBase.hpp

```diff
@@ -83,6 +83,9 @@ class JvmtiEnvBase : public CHeapObj<mtInternal> {
   static void leaving_dying_thread_env_iteration()  { --_dying_thread_env_iteration_count; }
   static bool is_inside_dying_thread_env_iteration(){ return _dying_thread_env_iteration_count > 0; }
 
+  static bool enable_virtual_threads_notify_jvmti();
+  static bool disable_virtual_threads_notify_jvmti();
```

Comment:
"disable" only seems to be used by the WB API. Is that expected?

Reply:
Yes, this is from the PR description:
> New function JvmtiEnvBase::disable_virtual_threads_notify_jvmti() which is needed for testing.
> It is used by the WhiteBox API.

Do you think a comment is needed?

Reply:
The function `JvmtiEnvBase::disable_virtual_threads_notify_jvmti()` is needed to test the `JvmtiEnvBase::enable_virtual_threads_notify_jvmti()`. To better test enabling we need to run it multiple times, so disabling is needed as well.